optimism/deployments/utils/dependencies.ts

```diff
@@ -34,6 +34,15 @@ export default {
     69: "0x93910D1D112AEA86E20c998948897b0cd37B3Ed9"
   },
 
+  // Prod SetTokens
+  MNY_ETH_TOKEN: {
+    10: {
+      staging: "0x93910D1D112AEA86E20c998948897b0cd37B3Ed9",    // dummy address
```

Comment:
@cgewecke I have to add a dummy address here for the tests to pass. Are there other ways?